polytope_server/common/staging/s3_staging.py

```diff
@@ -71,6 +73,7 @@ def __init__(self, config):
         )
 
     def create(self, name, data, content_type):
+        start = timeit.default_timer()
```

Comment:
You could use a decorator to wrap the function and avoid having 2 exit points. Alternatively, you could move the exit logic into a finally clause.

Reply:
For now I got rid of the log in the exception case as this should work for our current use case.